active/corsair.py

```diff
@@ -0,0 +1,56 @@
+
+"""
+
+This is an adaptation of Corsair Scan (https://github.com/Santandersecurityresearch/corsair_scan) as a ZAP plugin.
+This is delivered as an active scan plugin that will resend requests to all the sites captured in ZAP, injecting different origins. Then, it will read the value of Access-Control-Allow-Origin and based on that, it
```

Comment:
```suggestion
This script will resend requests to all the sites captured in ZAP, injecting different origins. Then, it will read the value of Access-Control-Allow-Origin and based on that, it
```